src/test/regress/expected/multi_insert_select.out

```diff
@@ -2212,7 +2212,7 @@ FROM
   table_with_defaults
 GROUP BY
   store_id, first_name, last_name;
--- Volatile function in default should be disallowed
+-- Volatile function in default should be disallowed - serial prototypes
```

Comment:
not sure what is meant by "serial prototypes"

Reply:
I meant to write serial pseudo-types. 😆